app/assets/javascripts/stores/users_store.es6.jsx

```diff
@@ -0,0 +1,23 @@
+(() => {
+  class UsersStore {
+
+    constructor() {
+      this.sidebar = true;
+      this.users = [];
+      this.bindListeners({
+        handleToggleSidebar: UsersActions.TOGGLE_SIDEBAR,
+        handleStoreUsers: UsersActions.STORE_USERS,
+      });
+    }
+
+    handleToggleSidebar() {
+      this.sidebar = !this.sidebar;
+    }
+
+    handleStoreUsers(users) {
+      this.users = users;
```

Comment:
This should be `this.users = response.users` and the parameter should be `response` instead of `users` now.